bucket/slash32.go

```diff
@@ -141,19 +137,35 @@ func (b *Slash32) truncate(truncatedSize int) {
 func (b *Slash32) ticker() {
 	ticker := time.NewTicker(time.Second)
 	for range ticker.C {
```

Comment:
I just switched order of truncating and adding credits, it felt like request should have a chance of getting a coin before the truncation, but I guess it's arbitrary..